src/models/spaces/populations/mouse-model/mouse-populations-model.ts

```diff
@@ -226,10 +226,13 @@ export const MousePopulationsModel = types
 
           buttons.push({
             title: "Scale",
+            type: "float-button",
```

Comment:
Marking a button as type `float-button` means we will handle it differently than the other buttons and try to position it somewhere floating in the right panel.